database/tool.gpt

```diff
@@ -3,32 +3,32 @@ Name: Database
 Description: Tools for interacting with a database
 Metadata: category: Capability
 Metadata: icon: https://cdn.jsdelivr.net/npm/@phosphor-icons/core@2/assets/duotone/database-duotone.svg
-Share Tools: Query, Exec
+Share Tools: Run Query, Exec Statement
 
 ---
-Name: Tables
-Description: List all tables in the SQLite database and return the results in markdown format
+Name: List Tables
+Description: List all tables in the SQLite database and return a JSON object containing the results
```

Comment:
> "the results in markdown format"

Just to double check - you didnt change the return format right? You are just updating the description to match reality?

Reply:
@cjellick Yes, I'm just updating it to match a change Darren made a while back. We were originally returning markdown formatted data. It's been JSON for a while at this point.